### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by ALobker**
Created the logger from assignment 1

### CODE REVIEW

File: core/src/com/group66/game/logging/FileLogger.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+public class FileLogger extends Logger {
+    private static FileHandle handle;
+
+    /**
+     * Constructor for a file logger
+     * @param mt the message type to write
+     */
+    public FileLogger(MessageType mt){
+        String date = DateFormatUtils.format(new Date(), "yyyyMMdd-HHmmss");
```
Comment (iiKoe) [inline core/src/com/group66/game/logging/FileLogger.java]: Change format to `yyyyMMddHHmmss` please, as that was what I put in the requirements.

I also put into the requirements: 
> The logger must provide timestamps in front of all the messages

and

> The logger should log in the line format: `[timestamp] LEVEL: message`

So could you move the timestamp to the abstract class, make the message as described in the requirements:
`[timestamp] LEVEL: message`
And just pass that to `write()`

Comment (ALobker) [inline core/src/com/group66/game/logging/FileLogger.java]: Fixed

**Comment by iiKoe** [comment]
The implementation of the type is different from that of the requirements.
Here a type is assigned to a Logger e.g. ConsoleLogger, in the requirements the main logger has maximum severity type that is shown:

> The logger should distinguish severity levels ERROR, WARNING, INFO, DEBUG
and and in the code specify the maximum severity level to be logged. This means
that when WARNING is selected, ERROR and WARNING messages will be logged,
and INFO and DEBUG messages will be discarded.

This means a message that is send will have the form:
`logger.log(MessageType.Warning, "This is a warning");`

**Comment by iiKoe** [comment]
I still want to change the file location, as for me they now appear in my home folder :(
Will do that later

**Pull request merged**